agignore

```diff
@@ -1,3 +1,4 @@
 log
 tags
 tmp
+/.git/
```

Comment:
removed the leading slaash, kept the trailing to make sure we're ignoring a directory and not a file named `.git`. Should probably update log and tmp similarly, but I'll do that in a follow-up